include/platform/clinician_input.h

```diff
@@ -2,10 +2,10 @@
 * This file contains the getter for the clinicina inputs
 */
 
-#include "common.h"
+#include <platform/common.h> 
```

Comment:
Would @LaureMillet 's previous comment about C style convention of " vs <> apply here also?